inventory/all_projects/pemm

```diff
@@ -0,0 +1,4 @@
+[pemm_staging]
+cdh-test-pemm1.princeton.edu
+[pemm_production]
+cdh-pemm1.princeton.edu
```

Comment:
we're just about to decommission the pemm servers, remember? (I think you were going to wait a week before shutting them down; you already turned off the public view)